docs/pages/index.js

```diff
@@ -39,7 +39,7 @@ const useStyles = makeStyles(
       flex: '1 0 100%',
     },
     hero: {
-      paddingTop: theme.spacing(8),
+      paddingTop: theme.spacing(8) + 30,
```

Comment:
If it was going to HEAD, we could have asked for a const, but since it's for a legacy branch, YOLO 👍